src/test/java/teammates/test/cases/browsertests/FeedbackMcqQuestionUiTest.java

```diff
@@ -59,10 +59,10 @@ public void testNewQuestionFrame() {
         feedbackEditPage.clickNewQuestionButton();
         feedbackEditPage.selectNewQuestionType("MCQ");
         assertTrue(feedbackEditPage.verifyNewMcqQuestionFormIsDisplayed());
-        if (!feedbackEditPage.isElementSelected("generateMcqOptionsCheckbox-1")) {
+        if (!feedbackEditPage.isElementSelected("generateMcqOptionsCheckbox--1")) {
             feedbackEditPage.clickGenerateMcqOptionsCheckbox(-1);
         }
-        assertFalse(feedbackEditPage.isElementPresent("mcqChoiceTable--1"));
+        assertFalse(feedbackEditPage.isElementVisible("mcqChoiceTable--1"));
 
         feedbackEditPage.clickDiscardChangesLinkForNewQuestion();
 
```

Comment:
Test runner stops here. No new question frame is opened in test browser. Any thoughts?

Reply:
I tried running your test locally; the new question cannot be clicked because the discard changes confirmation dialog is still open.